infra/multi_cluster_service.tf

```diff
@@ -19,7 +19,7 @@ resource "google_project_iam_member" "my_service_account_role_network_viewer" {
   role    = "roles/compute.networkViewer"
   member  = "serviceAccount:${google_service_account.my_service_account.email}"
   depends_on = [
-    module.enable_google_apis
+    module.enable_base_google_apis
```

Comment:
the name of the file implies that resources should be related to multi cluster. maybe it worth to move this resource declaration to another file?

Reply:
Another good thought.
Judging from [this comment](https://cloud.google.com/kubernetes-engine/docs/how-to/multi-cluster-services#authenticating), I added this `google_project_iam_member` resource because of [this cloud.google.com documentation about multi-cluster services](https://cloud.google.com/kubernetes-engine/docs/how-to/multi-cluster-services#authenticating). But that cloud.google.com doc also says:
> You do not need to do any steps in this section if you registered a cluster to a fleet with Workload Identity enabled.

So maybe we actually don't need it (and I added it by mistake).
I'll test if we can remove it altogether.
But I'd say this is out-of-scope, so I've started [a separate pull-request](https://github.com/GoogleCloudPlatform/terraform-ecommerce-microservices-on-gke/pull/53).